src/documents/ChangeApplierAPI.md

```diff
@@ -248,7 +367,7 @@ at the time the component is constructed. Note that programmatically attached li
 The listener is notified after the change (or set of coordinated changes) has already been applied to the model - it is too late to affect this process and so this event is not _preventable_. The signature for these listeners is
```

Comment:
A new line is missing btw this paragraph and the paragraph in front probably because the previous paragraph is using html `<div>`. Try adding a `<br />` at the end.

I'm also confused with the statement of "it is too late to affect this process". What process is too late to be affected? The model change process? Are listeners attached via the declarative way able to stop or prevent the model change by returning a false or something?